mailbox/store/src/main/java/org/apache/james/mailbox/store/search/MessageSearches.java

```diff
@@ -621,37 +623,60 @@ private boolean after(Date date1, Date date2, DateResolution res) {
     }
 
     private String createDateString(Date date, DateResolution res) {
```

Comment:
Not the code modified by this PR. I gonna change this, but please conciderI may not be able to sanitise all the search in this PR...